Apps/LocalAI/docker-compose.yml

```diff
@@ -0,0 +1,131 @@
+version: '3.6'
+name: localai
+services:
+  api:
+    environment:
+      MODELS_PATH: /models
+      PRELOAD_MODELS: '[{"url": "https://raw.githubusercontent.com/go-skynet/model-gallery/main/gpt4all-j.yaml","name": "gpt4all-j"}]'
```

Comment:
```suggestion
      PRELOAD_MODELS: '[{"url": "https://raw.githubusercontent.com/go-skynet/model-gallery/main/gpt4all-j.yaml","name": "gpt-3.5-turbo"}]'
```

In order to make the UI working, the model needs to be installed as "gpt-3.5-turbo". Chatbot-UI filters out the available models to match the OpenAI names